addons/delete-pop/addon.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Remove default sounds in sprites",
```

Comment:
Everyone who suggested this addon explicitly mentioned "Pop". It should be included in the addon name.

Reply:
```suggestion
  "name": "Remove default Pop sound in sprites",
```

Reply:
or:

```suggestion
  "name": "Remove default "pop" sound in sprites",
```